src/any_llm/providers/gemini/utils.py

```
@@ -307,13 +308,14 @@ def _convert_messages(
-                content_json = json.loads(message["content"])
-                part = types.Part.from_function_response(name=name, response=_normalize_tool_response(content_json))
-                formatted_messages.append(types.Content(role="function", parts=[part]))
-            except json.JSONDecodeError:
-                part = types.Part.from_function_response(name=name, response={"result": message["content"]})
-                formatted_messages.append(types.Content(role="function", parts=[part]))
+            content = message["content"]
+            if isinstance(content, str):
+                try:
+                    content = json.loads(content)
+                except json.JSONDecodeError:
+                    pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(json.JSONDecodeError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(json.JSONDecodeError): ...`

(SIM105)